### PR TITLE
[MINOR] docs(jdbc): polish jdbc document

### PR DESCRIPTION
**Pull request by FANNG1**
### What changes were proposed in this pull request?
1. correct jdbc document capacity 
2. polish docuement

### Why are the changes needed?
There're some errors in JDBC catalog

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
just document

### CODE REVIEW

File: docs/jdbc-mysql-catalog.md [hunk 1]
```diff
@@ -24,10 +24,12 @@ Gravitino saves some system information in schema and table comment, like `(From
...
 
-- Supports metadata management of MySQL (5.6, 5.7, 8.0).
+- Gravitino catalog corresponds to the MySQL instance.
+- Supports metadata management of MySQL (5.7, 8.0).
 - Supports DDL operation for MySQL databases and tables.
-- Doesn't support table index operations.
-- Doesn't support setting certain column properties, such as default value and check constraints.
+- Supports table index.
+- Supports column default value and auto-increment.
+- Supports manage MySQL table features though table properties, like using `engine` to set MySQL storage engine.
```
Comment (jerryshao) [inline docs/jdbc-mysql-catalog.md:32]: “Supports managing...”

File: docs/jdbc-mysql-catalog.md [hunk 2]
```diff
@@ -76,9 +78,9 @@ Please refer to [Manage Metadata Using Gravitino](./manage-metadata-using-gravit
 
 - Gravitino table corresponds to the MySQL table.
 - Supports DDL operation for MySQL tables.
-- Doesn't support setting certain column properties, such as default value and check constraints.
-- Doesn't support index definition.
-- Doesn't support table property settings.
+- Supports index.
+- Support column default value and auto-increment.
+- Supports manage MySQL table features though table properties, like using `engine` to set MySQL storage engine.
```
Comment (jerryshao) [inline docs/jdbc-mysql-catalog.md:83]: Supports managing...

File: docs/jdbc-mysql-catalog.md (hunk 1 above)
Comment (FANNG1) [reply docs/jdbc-mysql-catalog.md]: done

File: docs/jdbc-mysql-catalog.md (hunk 2 above)
Comment (FANNG1) [reply docs/jdbc-mysql-catalog.md]: done

**Review by jerryshao — APPROVED** [review]

**Review by jerryshao — APPROVED** [review]

**Pull request merged**